### PR TITLE
fix mime component dependency for symfony 5

### PR DESCRIPTION
**Pull request by ph538834**
The uploadable extension uses the deprecated mimetypeguesser class.
this class is gone in sf 5 and the usage no longer possible.
i've forked from the 1.4.2 because master is broken.

### CODE REVIEW

File: Uploadable/MimeTypeGuesserAdapter.php [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Symfony\Component\Mime\MimeTypes;
 use Gedmo\Uploadable\MimeType\MimeTypeGuesserInterface;
 
 class MimeTypeGuesserAdapter implements MimeTypeGuesserInterface
 {
     public function guess($filePath)
     {
-        return MimeTypeGuesser::getInstance()->guess($filePath);
+        $mimeTypes = new MimeTypes();
```
Comment (jorrit) [inline Uploadable/MimeTypeGuesserAdapter.php]: MimeTypes has a heavy constructor. Perhaps you could call `MimeTypes::getDefault()` or add MimeTypes as a constructor argument to have it injected.

Comment (ph538834) [reply Uploadable/MimeTypeGuesserAdapter.php]: Thank you for your comment.
Changed to the static getDefault() Variant. Dont know how to do it with constructor, i assume i would have to configure this as a service? 
Are there relevant advantages?

Comment (jorrit) [reply Uploadable/MimeTypeGuesserAdapter.php]: I don't think there are important advantages. Maybe someone configures something specific in their application. But if that is the case, they should open a PR :)

**Comment by antishov** [comment]
Thank you!

**Pull request merged**